install/letsencrypt.bash

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+userName=$1
+FQDN=$2
+configFile=/etc/letsencrypt/"$FQDN".cli.ini
+
+scriptDir="$(dirname $0)"
+cd $scriptDir
+# Read config file with paths to WP-installs and usernames
+source ../conf
+
+# Create configfolder+file if letsencrypt hasn't run yet
+mkdir -p /etc/letsencrypt
+touch $configFile
+
+# Create a config file for domain
+cat > $configFile << EOF
```

Comment:
Entirely unrelated: EOF sort of sounds like a onomatopoeic word for a dog's bark.